<?xml version="1.0"?>

<launch>

  <!-- Run the adapters -->
  <group ns="adapter">
    <node name="back_IMU_adapter" pkg="group_project" type="back_IMU_adapter.py" output="screen"/>
    <node name="lla_IMU_adapter" pkg="group_project" type="lla_IMU_adapter.py" output="screen"/>
    <node name="lua_IMU_adapter" pkg="group_project" type="lua_IMU_adapter.py" output="screen"/>
    <node name="rla_IMU_adapter" pkg="group_project" type="rla_IMU_adapter.py" output="screen"/>
    <node name="rt_IMU_adapter" pkg="group_project" type="rt_IMU_adapter.py" output="screen"/>
    <node name="rua_IMU_adapter" pkg="group_project" type="rua_IMU_adapter.py" output="screen"/>
  </group>

</launch>